pkg/operator/operator/validations.go

```diff
@@ -744,3 +789,54 @@ func getValidationK8sResources() ([]kunstructured.Unstructured, *kresource.Quant
 
 	return virtualServices, maxMem, err
 }
+
+func validateDockerImagePath(image string) error {
+	if _, err := cr.ValidateImageVersion(image); err != nil {
+		return err
+	}
+
+	dockerAuth := dockerlib.NoAuth
+	if regex.IsValidECRURL(image) {
+		operatorID, _, err := config.AWS.GetCachedAccountID()
+		if err != nil {
+			return err
+		}
+		registryID := aws.GetAccountIDFromECRURL(image)
+
+		if strings.Compare(operatorID, registryID) != 0 {
+			return ErrorRegistryAccountIDMismatch(registryID, operatorID)
+		}
+
+		ecrAuthConfig, err := config.AWS.GetECRAuthConfig()
+		if err != nil {
+			return err
+		}
+
+		dAuth, err := dockerlib.EncodeAuthConfig(dockertypes.AuthConfig{
+			Username:      ecrAuthConfig.Username,
+			Password:      ecrAuthConfig.AccessToken,
+			ServerAddress: ecrAuthConfig.ProxyEndpoint,
+		})
+		if err != nil {
+			return err
+		}
+		dockerAuth = dAuth
+	}
+
+	client, err := dockerclient.NewEnvClient()
+	if err != nil {
+		return err
+	}
+
+	if err := dockerlib.CheckImageAccessible(client, image, dockerAuth); err != nil {
+		// the concrete type of docker client errors is string
+		// it's just better to put them on a new line
+		// because they are quite verbosy
+		return ErrorDockerImageInaccessible(
```

Comment:
Since there is no wrapping here, `CheckImageAccessible()` can return the `ErrorDockerImageInaccessible` error